tfe/resource_tfe_variable_migrate_test.go

```diff
@@ -1,45 +0,0 @@
-// Copyright (c) HashiCorp, Inc.
-// SPDX-License-Identifier: MPL-2.0
-
-package tfe
```

Comment:
TO-DO: remove file name `resource_tfe_variable_migrate_test` from `.golangci.yml`

Reply:
!! Good catch, thank you!